doc/digital-receipt/Austria/digital-receipt-austria.md

```diff
@@ -0,0 +1,35 @@
+---
+slug: /poscreators/middleware-doc/digital-receipt/austria
+title: 'Digital Receipt: Austria'
+---
+
+# Compliance AT
+
+Vorschriften an den digitalen Beleg in Österreich:
+
+Ein elektronischer Beleg iSd § 132a Abs. 1 BAO ist ein Beleg, der in einem elektronischen Format ausgestellt und empfangen bzw. unmittelbar für den die Barzahlung Leistenden verfügbar ist. Er kann z.B. mittels E-Mail, als E-Mail-Anhang oder Web-Download, in einem elektronischen Format (z.B. als PDF- oder Textdatei), aber auch in einem strukturierten Dateiformat (z.B. HTML) ausgestellt werden. Der Belege muss unmittelbar im Zusammenhang mit der Barzahlung durch die Registrierkasse erstellt und signiert werden, sowie in der Folge tatsächlich in den Verfügungsbereich des Belegempfängers gelangen. Technisch bedingte Verzögerungen wie z.B. bei der E-Mail-Zustellung oder dem Upload zu einem Server sind unbeachtlich, wenn die elektronischen Belege sofort im DEP sowie zeitnah in einem revisionssicheren Speicher (vgl. Abschnitt 3.2.1.4. Sicherung des DEP) gespeichert und für eine Überprüfung durch ein Organ der Finanzverwaltung verfügbar sind. Ob dies elektronisch (per Email, Ablesen mittels Smartphone, usw.) oder in Papierform erfolgt, bleibt dem Belegaussteller überlassen. Die Übermittlung ist eine Bringschuld des Unternehmers, eine bloße Einräumung der Möglichkeit des Ansehens und Abfotografierens des auf einem Bildschirm angezeigten Beleginhaltes erfüllt nicht die Belegerteilungspflicht. Eine spezielle Form der elektronischen Übertragung ist nicht vorgeschrieben. Ob die Übertragung in den Verfügungsbereich des Belegempfängers durch Aktivität des Leistungserbringers oder den die Bezahlung Leistenden erfolgt, ist für die Erfüllung der Belegerteilungspflicht unerheblich. Diese Art der Belegerteilung ist im elektronischen Aufzeichnungssystem zu dokumentieren.
+
+fiskaltrust hat ein Gutachten in Auftrag gegeben, welches das Ziel verfolge, zu untersuchen, ob verschiedene Möglichkeiten der digitalen Belegerstellung aus technischer Sicht der RKSV entsprechen. In diesem Gutachten wurden diese Ansätze sowohl prinzipiell an Hand des Konzeptes als auch an Hand von konkreten Implementierungen untersucht.
```

Comment:
fiskaltrust has commissioned an external assessment with the aim of examining whether various options for digital document creation correspond to the RKSV from a technical point of view. In this report, these approaches were examined both in principle on the basis of the concept and on the basis of concrete implementations.